src/Math/Extensions.cs

```diff
@@ -62,226 +47,173 @@ public static double Clamped(this double value, double min, double max)
-        }
+        public static float Acos(this float f) => (float)System.Math.Acos(f);
 
         /// <summary>
         /// Returns the angle whose sine is the specified number.
         /// </summary>
-        /// <param name="f"></param>
-        /// <returns></returns>
-        public static float Asin(this float f)
-        {
-            return (float)System.Math.Asin(f);
-        }
+        public static float Asin(this float f) => (float)System.Math.Asin(f);
 
         /// <summary>
         /// Returns the angle whose tangent is the specified number.
         /// </summary>
-        /// <param name="f"></param>
-        /// <returns></returns>
-        public static float Atan(this float f)
-        {
-            return (float)System.Math.Atan(f);
-        }
+        public static float Atan(this float f) => (float)System.Math.Atan(f);
 
         /// <summary>
         /// Returns the square root of the specified number.
         /// </summary>
-        /// <param name="f"></param>
-        /// <returns></returns>
-        public static float Sqrted(this float f)
-        {
-            return (float)System.Math.Sqrt(f);
-        }
+        public static float Sqrted(this float f) => (float)System.Math.Sqrt(f);
 
         /// <summary>
         /// Returns a specified number raised to the specified power.
         /// </summary>
-        /// <param name="b"></param>
-        /// <param name="power"></param>
-        /// <returns></returns>
-        public static float Powed(this float b, float power)
-        {
-            return (float)System.Math.Pow(b, power);
-        }
+        public static float Powed(this float b, float power) => (float)System.Math.Pow(b, power);
 
         /// <summary>
         /// Returns the lowest integral number higher than or equal to the specified number.
         /// </summary>
-        /// <param name="f"></param>
-        /// <returns></returns>
-        public static int CeiledToInt(float f)
-        {
-            return (int)System.Math.Ceiling(f);
-        }
+        public static int CeiledToInt(float f) => (int)System.Math.Ceiling(f);
 
         /// <summary>
         /// Returns the highest integral number lower than or equal to the specified number.
         /// </summary>
-        /// <param name="f"></param>
-        /// <returns></returns>
-        public static int FlooredToInt(float f)
-        {
-            return (int)System.Math.Floor(f);
-        }
+        public static int FlooredToInt(float f) => (int)System.Math.Floor(f);
 
         /// <summary>
         /// Returns the integral number closest to the specified number.
         /// </summary>
-        /// <param name="f"></param>
-        /// <returns></returns>
-        public static int RoundedToInt(float f)
-        {
-            return (int)System.Math.Round(f);
-        }
+        public static int RoundedToInt(float f) => (int)System.Math.Round(f);
+
+        #endregion
+
+        #region Double math
```

Comment:
We have power functions like `Cubed` grouped by method, while trigonometry is grouped by float/double. Please make consistent.